src/window-service.js

```diff
@@ -177,6 +178,27 @@
             mainWindow.addEventListener('closed', closedEvent);
         }
 
+        snapToScreenBounds(targetWindow) {
+            fin.desktop.System.getMonitorInfo((info) => {
+                targetWindow.getBounds((bounds) => {
+
+                    if (bounds.left < info.virtualScreen.left) {
+                        bounds.left = info.virtualScreen.left;
+                    } else if (bounds.left + bounds.width > info.virtualScreen.right) {
+                        bounds.left = info.virtualScreen.right - bounds.width;
```

Comment:
I take it you can't move `bounds.right` and have that shunt the whole window across?